### ISSUE TITLE
Signed binaries

### ISSUE THREAD

**Issue by abrobston** [fsprojects/FSharp.Compiler.CodeDom#3]
Is it possible to sign binaries in the NuGet package?  That would make it easier to use the binary from another project that will be signed (and, therefore, strong-named).

**Comment by dsyme** [comment]
Yes, this should be added. There's a discussion going on about this here: https://github.com/fsprojects/FSharp.Compiler.CodeDom/pull/6#issuecomment-94792936, please assist :)
